__init__.py

```diff
@@ -9,6 +9,7 @@
 from SR2ML.src import MaintenanceModel
 from SR2ML.src import MCSSolver
 from SR2ML.src import MarginModel
+from SR2ML.src import basicEventScheduler
```

Comment:
I would recommend to use upper case for each leading character when define a class. That is: change  basicEventScheduler to  BasicEventScheduler

Reply:
renamed